compressor/offline/django.py

```diff
@@ -26,7 +26,7 @@ def handle_extendsnode(extendsnode, block_context=None, original=None):
                   extendsnode.nodelist.get_nodes_by_type(BlockNode))
     block_context.add_blocks(blocks)
 
-    context = Context(settings.COMPRESS_OFFLINE_CONTEXT)
+    context = Context(dict(settings.COMPRESS_OFFLINE_CONTEXT))
```

Comment:
We're probably missing tests/features here, but this PR does fix a bug anyway, so I'm going to merge it.